chevah/compat/nt_filesystem.py

```diff
@@ -264,10 +286,10 @@ def getSegmentsFromRealPath(self, path):
 
         path = text_type(path)
 
-        target = os.path.abspath(path.replace('/', '\\')).lower()
+        target = os.path.normpath(path.replace('/', '\\')).lower()
```

Comment:
`os.path.abspath` is buggy when the current path is defined as UNC or drive path...so we try to avoid it.